src/components/LuxAutocompleteInput.vue

```diff
@@ -163,6 +163,7 @@ export default {
       // Is the data given by an outside ajax request?
       if (this.isAsync) {
         this.isLoading = true
+        this.isOpen = true
```

Comment:
could we pull `this.isOpen = true` out of the conditional, since it happens on both of the conditional branches?